runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -248,7 +248,7 @@ public static CorfuRuntimeParametersBuilder builder() {
         /*
          * Period of time in ms to sleep before next cycle, when poller gets no new data changes.
          */
-        private int streamingPollingIdleWaitTimeMs = 5;
+        private int streamingPollingIdleWaitTimeMs = 50;
```

Comment:
RIP. This is the wrong parameter. Should change `streamingNotificationBlockingTimeMs` and remove this one.